docs/contents/components/media-and-icons/avatar/accessibility.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: Avatar
+description: "`Avatar` is a component that displays a profile picture or an icon with initials representing a user."
+package_name: "@yamada-ui/avatar"
+with_description: true
+tab: Accessibility
+---
+
+## ARIA Roles and Attributes
+
+| Component     | Role and Attributes        | Usage                                                                                               |
+| ------------- | -------------------------- | --------------------------------------------------------------------------------------------------- |
+| `AvatarName`  | `role="img"`               | Indicates that it is an image when the image set in `src` is not yet loaded or if `src` is not set. |
```

Comment:
The `AvatarName` is not provided. It is used internally.

Reply:
indeed

Reply:
`AvatarName` was summarised in `Avatar`.

Reply:
deleted `AvatarName`